packages/trpc/server/routers/viewer/apps/_router.tsx

```diff
@@ -124,4 +127,53 @@ export const appsRouter = router({
         input,
       });
     }),
+  // TODO: @Hariom should we move these elsewhere?
```

Comment:
@hariombalhara should we move this to a new router? Or is it okay here for now?

Reply:
Making a new router is better IMHO as it's not an app.

Reply:
Will do :)

Reply:
Also, while you are at it, I think this approach would be cleaner to follow(avoids duplication of  UNSTABLE_HANDLER_CACHE code) https://github.com/calcom/cal.com/blob/main/packages/app-store/routing-forms/trpc/_router.ts#L48

Reply:
Will make it reusable :)

Reply:
I think we should actually leave it as for now. @keithwillcode shall we create a seperate PR to improve this across all existing routers? 

Will still move these procedures to a new router however